src/Umbraco.Tests.Integration/Services/ContentServiceEventTests.cs

```diff
@@ -7,24 +7,29 @@
 using Umbraco.Core.Persistence.Repositories.Implement;
 using Umbraco.Core.Services;
 using Umbraco.Core.Services.Implement;
-using Umbraco.Tests.Common.Builders;
+using Umbraco.Tests.Integration.Testing;
 using Umbraco.Tests.TestHelpers.Entities;
 using Umbraco.Tests.Testing;
 
-namespace Umbraco.Tests.Services
+namespace Umbraco.Tests.Integration.Services
 {
     [TestFixture]
     [UmbracoTest(Database = UmbracoTestOptions.Database.NewSchemaPerTest,
         PublishedRepositoryEvents = true,
         WithApplication = true,
         Logger = UmbracoTestOptions.Logger.Console)]
-    public class ContentServiceEventTests : TestWithSomeContentBase
+    public class ContentServiceEventTests : UmbracoIntegrationTest
     {
+        private IContentTypeService _contentTypeService => GetRequiredService<IContentTypeService>();
+        private IContentService _contentService => GetRequiredService<IContentService>();
+        private ILocalizationService _localizationService => GetRequiredService<ILocalizationService>();
+        private IFileService _fileService => GetRequiredService<IFileService>();
```

Comment:
All of these do not follow the convension for private properties. Should be Title case

Reply:
Agreed, have updated.